middleware/src/main/java/br/ufrn/dimap/middleware/identification/lookup/ILookup.java

```diff
@@ -0,0 +1,38 @@
+package br.ufrn.dimap.middleware.identification.lookup;
+
+import br.ufrn.dimap.middleware.identification.AbsoluteObjectReference;
+import br.ufrn.dimap.middleware.remotting.impl.RemoteError;
+
+/**
+ * Implements a Lookup Service as part of the distributed object middleware. 
+ * This service provides an way to bind some name to an remote object absolute 
+ * reference, and other way to find this absolute reference given an name.
+ * 
+ * @author Yuri Alessandro Martins 
+ * @version 1.0
+ * @see AbsoluteObjectReference
+ * @see RemoteError
+ */
+public interface ILookup {
```

Comment:
Name the interface simply as Lookup, not ILookup. I've got a good link about this: https://stackoverflow.com/questions/2814805/java-interfaces-implementation-naming-convention/2814831

Reply:
done